articles/azure-node-health-checks/reviewing-results.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Interpreting results from Azure Node Health Check
+description: Interpreting results from Azure Node Health Checks (AzNHC) to validation.
+ms.service: azure-node-health-checks
+ms.topic: how to 
+author: rafaelsalas
+ms.author: rafaelsalas
+ms.date: 04/15/2024
+---
+
+# Interpreting Azure Node Health Check Results
+AzNHC prints results to both terminal and it's health log. The error messages produced by AzNHC gives are useful for determining the failed components. The following will guide you through understanding what a successful/failed test output looks like.
```

Comment:
This looks great!   Could we request a command line option to redirect ALL test output to log file (and/or stderr) and only output SUCCESS or FAILURE lines to stdout to make parsing a bit easier?

Reply:
Same with this one, we would have to track on the AzNHC repo. This one may not be as straight forward as the first but we can look into it.